### DISCUSSION TITLE
Can't run new Chia 1.2 on Mac OS.... gives malware error with no option to bypass.

### DISCUSSION THREAD

**Discussion by cvee72** [Chia-Network/chia-blockchain#7191] [Support]
Hi,

Just installed Chia 1.2 on Mac os... previous versions worked.
Get error :-
“Chia” can’t be opened because Apple cannot check it for malicious software.
This software needs to be updated. Contact the developer for more information.

There is no options in system settings to bypass this restriction.

**Comment by joelrb** [comment]
> Hi,
> 
> Just installed Chia 1.2 on Mac os... previous versions worked.
> Get error :-
> “Chia” can’t be opened because Apple cannot check it for malicious software.
> This software needs to be updated. Contact the developer for more information.
> 
> There is no options in system settings to bypass this restriction.

Go to your Mac's Preferences > Security & Privacy > General tab.. look down to Allow apps downloaded from:

Apple flags any software as potentially malicious that does not come from the App Store.

**Reply by cvee72** [reply]
Yes normally this works... but NOT for the Chia app.
For whatever reason, the error message says the software needs to be updated.

System Settings Privacy General does NOT show any app for me to approve... which is what normally happens in other apps cases.

**Comment by loppefaaret** [comment]
we had issues with the installers for macos - please redownload the installer, and it should work!

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
